### PR TITLE
tests: rbd/rgw adapt testinfra for jewel

### PR DESCRIPTION
**Pull request by guits**
- the rbd-mirror unit systemd name is not the same when running jewel vs
luminous.
- servicemap is not available on jewel.

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -21,6 +21,18 @@ def node(host, request):
...
     osd_auto_discovery = ansible_vars.get("osd_auto_discovery")
     lvm_scenario = ansible_vars.get("osd_scenario") == 'lvm'
+    ceph_release_num = {
+      'dumpling': 0.67,
+      'emperor': 0.72,
+      'firefly': 0.80,
+      'giant': 0.87,
+      'hammer': 0.94,
+      'infernalis': 9,
+      'jewel': 10,
```
Comment (leseb) [inline tests/conftest.py]: I think we can start at Jewel and forget previous releases.

File: tests/conftest.py [hunk 2]
```diff
@@ -42,6 +54,12 @@ def node(host, request):
     if node_type == "nfss" and ceph_stable_release == "jewel":
         pytest.skip("nfs nodes can not be tested with ceph release jewel")
 
+    if request.node.get_marker("from_luminous") and ceph_release_num[ceph_stable_release] < ceph_release_num['luminous']:
+        pytest.skip("This test is valid only for release after luminous")
```
Comment (leseb) [inline tests/conftest.py]: `This test is only valid for releases starting from Luminous and above.`

File: tests/conftest.py [hunk 3]
```diff
@@ -42,6 +54,12 @@ def node(host, request):
...
+
+    if request.node.get_marker("before_luminous") and ceph_release_num[ceph_stable_release] >= ceph_release_num['luminous']:
+        pytest.skip("This test is valid only for release before luminous")
```
Comment (leseb) [inline tests/conftest.py]: ditto

**Comment by guits** [comment]
jenkins test luminous-ansible2.3-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.3-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.3-xenial_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.3-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.3-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.3-centos7_cluster

**Comment by guits** [comment]
jenkis test luminous-ansible2.3-docker_cluster

**Comment by guits** [comment]
jenkis test luminous-ansible2.3-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.3-docker_cluster

**Comment by guits** [comment]
jenkins test all

**Comment by guits** [comment]
jenkins test luminous-ansible2.3-bluestore_dmcrypt_journal

**Comment by guits** [comment]
jenkins test luminous-ansible2.3-dmcrypt_journal_collocation

**Comment by guits** [comment]
jenkins test luminous-ansible2.3-journal_collocation

**Comment by guits** [comment]
jenkins test luminous-ansible2.3-journal_collocation

**Comment by guits** [comment]
jenkins test this please

**Pull request merged**